tools/run_danesfield.py

```diff
@@ -490,6 +490,24 @@ def main(args):
              'texture-mapping',
              cmd_args)
 
+    #############################################
+    # 3D Tiles Generation
+    #############################################
+    tiler_outdir = os.path.join(working_dir, 'tiler')
+    input_tiler = glob.glob(os.path.join(texture_mapping_outdir, "*.obj"))
+    utm_zone = 17        # jacksonville dataset
+    utm_hemisphere = "N"
```

Comment:
I don't think we should have to hardcode anything here as this information should be coming through the configuration files (or we should be able to derive that info from what's in the config).  (Perhaps it's in the VisSat config @jacobderosa ?)

Reply:
I agree. However this is what we have so far afaik. We'll need code to get the the zone from images (or point clouds) and propagate it here.

Reply:
UTM information is included in the VisSat config file. Though that won't be helpful if a user already has a point cloud and doesn't need to run the VisSat step

Reply:
Full georeference information should be read from the image or point cloud files and then propagated to the mesh (building model). I think even a vissat config file is a hack. What happens if you process an image from a different UTM zone?

Reply:
@mleotta We should discuss this in our meeting to decide what we do.